### DISCUSSION TITLE
Scale prometheus pod

### DISCUSSION THREAD

**Discussion by mgorzkow** [prometheus-operator/kube-prometheus#1988] [General] [answered]
I have two pods prometheus-k8s-0 and prometheus-k8s-1. How can i scale them up to 3 pods?

**Comment by metalmatze** [comment, marked as answer]
By going to the `replicas` key in the Prometheus custom resource and setting it to `3`. 
`kubectl edit prometheus -n monitoring k8s` or changing the jsonnet/YAML.
